src/app/jscaip/MoveCoordToCoord.ts

```diff
@@ -6,6 +6,11 @@ import { MoveWithTwoCoords } from './MoveWithTwoCoords';
 
 export abstract class MoveCoordToCoord extends MoveWithTwoCoords {
 
+    public static equals<T extends MoveCoordToCoord>(first: T, second: T): boolean {
```

Comment:
hmm pourquoi l'avoir en statique ? si t'as un `first: T`, tu peux toujours faire `first.equals(second)` !

Reply:
Arh, j'avais un this.equals(other) qui marchait pas mais du coup nique toi, other.equals(this) et voilà, plus besoin de trick surrérogatocodogêne ! (comment on dit en un mot: qui génère du code non nécessaire ?)

Reply:
on appelle cela de la dette technique éventuellement ? 

Et euh, si tu fais `super.equals` dans `CoerceoNormalMove.equals`, tu accèdes bien à cette méthode, donc no need for static !

Reply:
mmh, j'ai eu des foirage, mais là ça as l'air bin bon !